src/area_utils.py

```diff
@@ -127,7 +127,7 @@ def load_raster(
     in_raster_basename = os.path.basename(in_raster)
 
     with rio.open(in_raster) as src:
-        if src.meta["crs"] == "EPSG:4326":
+        if src.meta["crs"] == "EPSG:4326" or True:
```

Comment:
Isn't this if statement always going to be True?

Reply:
@hannah-rae can you give a hint of the use case of `True` here?